test/controllers/recipe_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class RecipeControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
Rails generates spec files for minitest when using generators, so we may want to re-assess adding these now if we plan on using `Rspec` for our testing purposes.

Reply:
We should delete these files in a different PR.